src/3D/L3DAnim.cpp

```diff
@@ -13,7 +13,7 @@
 #include <glm/gtx/matrix_interpolation.hpp>
 #include <spdlog/spdlog.h>
 
-#include "Common/FileSystem.h"
```

Comment:
@bwrsandman 
`#include "FileSystem.h"` feels a bit too open. Should we go for `#include "file_utils/FileSystem.h"`?